reef_control_pid: {
    kp: 0.8, deadzone: 0.1, max_vel: 1.5, #params for sigmoid
    center_point: 1.0, alpha: 0.4, #params for sigmoid
    uP: 0.9, uI: 0.01, uD: 0.02, #body_frame x velocity
    vP: 0.7, vI: 0.01, vD: 0.02, #body_frame y velocity
    wP: 1.0, wI: 0.1, wD: 0.0, uvtau: 0.15, #body_frame z velocity
    dP: 1.8, dI: 0.1, dD: 0.0, nedtau: 0.15, #body_frame z
    yawP: 1.08, yawI: 0.08, yawD: 0.01, yawtau: 0.15, #yaw
    xIntegrator: 1,
    uIntegrator: 1,
    hover_throttle: 0.57,

    max_roll: 0.25,
    max_pitch: 0.25,
    max_yaw_rate: 2.0,
    max_u: 1.0,
    max_v: 1.0,
    max_w: 1.0,
    max_d: 1.0,

    face_target: false,
    fly_fixed_wing: false,
    gaussian_offset: 0.0
}
